site/static/examples/multikueue/tekton-pipelinerun-example.yaml

```
@@ -0,0 +1,114 @@
+      operator: Exists
+
+---
+# 3. Cluster Queue that can use the worker cluster
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ClusterQueue
+metadata:
+  name: "tekton-queue"
+spec:
+  namespaceSelector: {} # match all namespaces
+  resourceGroups:
+  - coveredResources: ["cpu", "memory"]
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)